lib/food_truck.rb

```diff
@@ -0,0 +1,24 @@
+class FoodTruck
+  attr_reader :name, :inventory, :stock
+
+  def initialize(name)
+    @name = name
+    @inventory = {}
+  end
+
+  def check_stock(item)
+    if @inventory == {}
+      return 0
+    else
+      return @inventory[:item]
+    end
+  end
+
+  def stock(item, amount)
+    @inventory[item] += amount
```

Comment:
This would only work if an item already exists in the @inventory hash.  But, since this method is the way that we get things into that hash.... that will never work. You are trying to do math on `nil`